l2geth/core/vm/instructions.go

```diff
@@ -963,3 +963,9 @@ func makeSwap(size int64) executionFunc {
 		return nil, nil
 	}
 }
+
+// OVM opcodes
```

Comment:
Perhaps out of scope of this PR but we should probably migrate this to `L1TxOrigin` throughout the entire codebase so its 1:1 with the contracts. Would be a big diff with a find and replace

Reply:
We actually call it L1MessageSender in the contracts too now.